gems/sorbet-runtime/lib/types/private/methods/_methods.rb

```diff
@@ -300,10 +298,36 @@ def self.run_all_sig_blocks
     end
   end
 
+  # a module A such that:
+  #   if A is included or extended by a module B,
+  #   then we want B to have the hooks installed on itself
+  # should extend this module.
+  #
+  # note that the arg is by default nil so it doesn't conflict with `Chalk::Tools::Concern` in pay-server.
```

Comment:
This seems like a gotcha. What do you mean "conflict"? The correctness of sorbet-runtime shouldn't depend on how pay-server or any other ruby codebase works.

Reply:
Actually, this should no longer be necessary because we're not adding new logic for `included` and `extended` for any modules. But I was running into issues earlier with the number of arguments `include` being optionally 0 if you `extend Chalk::Tools::Concern`, which defines its own DSL with a method called `included` that takes as few as 0 arguments but also takes a block.